src/open_inwoner/accounts/tests/test_profile_views.py

```diff
@@ -548,9 +551,14 @@ def test_eherkenning_user_updates_klant_api(self, m):
                 # `m` is overridden somewhere, which causes issues when `MockAPIReadPatchData.install_mocks`
                 # is run for the second time
                 with requests_mock.Mocker() as m:
-                    data = MockAPIReadPatchData().install_mocks_eherkenning(
+                    # Reset the email, otherwise we get errors that a user already exists
+                    # with the updated email
+                    data.eherkenning_user.email = original_email
```

Comment:
This could have been/become two tests with less hassle but it's your choice.

Reply:
Yeah it's becoming quite messy, I split it into two